common/quickstart/aws-docker-helper.sh

```diff
@@ -0,0 +1,31 @@
+# Helper for the AWS specific host, to run a species/dataset instance of the Knetminer Docker container with AWS logging.
```

Comment:
I'm gonna change name to this file, I've been using -helper for scripts that aren't supposed to usually be called by the end user (they're used inside the containers or by Knetminer developers).